src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header1 {
+  box-sizing: border-box;
+  display: flex;
+  max-width: 1200px;
+  min-width: 1024px;
```

Comment:
your header should be full-width
do not limit it
<img width="1440" alt="Screenshot 2022-07-28 at 12 19 57" src="https://user-images.githubusercontent.com/94289038/181470409-53e4d594-b00c-4cd6-a499-4d12df55df54.png">